core/templates/pages/story-editor-page/navbar/story-editor-navbar.directive.ts

```diff
@@ -101,6 +101,13 @@ angular.module('oppia').directive('storyEditorNavbar', [
                     );
                     break;
                   }
+                  if (summaries[idx].language_code !== 'en') {
```

Comment:
> Yeah, though I meant this PR to only deal with the backend validations. Should I add these also in this PR?

I think so, if it's not too much trouble -- might as well get it over with and keep the frontend + backend consistent. However, if there are difficulties I'm not thinking of, it's also fine to file a follow-up issue for this.

Reply:
Currently, I am using the exploration summary api service in the frontend to validate explorations, but for this I'd need to fetch the explorations themselves, for which I couldn't find a call that fetched multiple explorations at once, or an endpoint needs to be added that would return valid or not for all expIds in story.
Either way, a new controller and api service change is needed, which I thought might be too much for one PR, so is it fine to file another issue or do a follow up PR myself?

Reply:
Yep, OK to file an issue and do a follow-up PR for this. Just make sure to add a TODO here too that points to said issue.